### DISCUSSION TITLE
Options for Printing Bottom Open

### DISCUSSION THREAD

**Discussion by LoopBak01** [KD7LRJ/3D-LDS-Temples#12] [General]
I'd like to print some of these with the bottom layers removed, however after a few prints I've noticed that the bottom layers exist in upper layers as well, creating holes in the print, and sometimes causing other problems. I've tried negating the z-axis as well to lower the print but I feel like that removes some of the essential details of the lower layers. Any thoughts or suggestions on how to do this? It would be nice if there was an option to just remove the bottom layers (the actual base of the print). Thoughts or suggestions?

**Comment by KD7LRJ** [comment]
I'm not sure I follow what you're trying to explain. If you just want to remove some of the bottom layers, most slicers have the ability to simply cut a model at whatever layer height you like and discard the unwanted portion.

I don't understand "the bottom layers exist in upper layers as well" at all and am completely lost on this part.

Also, what do you mean by the "base of the print"? Is there a temple model with an extra flat base piece attached to it that you don't want to include in the print? I've certainly printed some with added bases so the pillars of entry areas, etc. don't break off, but I don't think any of the ones here have such bases attached.

**Reply by OkinawJapanTemple** [reply]
Hey so I am a member of the church and living in Okinawa and the temple is compete and dedicated. So I was wondering if you could make a stl. file of the temple. There is a website released from the church that has a video of a 3d model, so if you could make it really exact like you normally do and maybe if you have time put the Okinawa Japan Temple on a Priority. Thanks if you can and thanks if you can't.

**Reply by KD7LRJ** [reply]
@hlyman Is the one who does the CAD modeling for all of the files in this collection so far. I don't know his schedule.

**Comment by LoopBak01** [comment]
Hey thanks for the response. Sorry I wasn't very clear.

I'm using cura to slice the Portland temple. I remove all bottom layers so
the setting is zero, and use a triangle infill (5% total infill). If you
preview the sliced model you'll see there are no bottom layers and you can
see the infill when viewing from below. (I do this to more easily fish a
string of lights through.) If you were to look at some of the walls further
up on the model where the walls meet the roof you'll see under the overhang
there are gaps.

Now if you slice it and keep the bottom layers,  you'll see the gaps under
those overhang roof parts are now filled in. Cura, shows those are bottom
layers as well. What I was referring to as the base are the bottom layers
that would make the bottom of the temple, the default 4 layers, that are
first printed on the bed.

So by removing the bottom layers on the print other "bottom layers" are
removed too that can affect the quality of the print. I'd like to find a
way to just remove the very bottom layers printed on the bed but keep the
bottom layers printed everywhere else on the print.

I hope this makes more sense. Thanks again for the help.



On Tue, Dec 27, 2022 at 8:37 PM Douglas B. McKay ***@***.***>
wrote:

> I'm not sure I follow what you're trying to explain. If you just want to
> remove some of the bottom layers, most slicers have the ability to simply
> cut a model at whatever layer height you like and discard the unwanted
> portion.
>
> I don't understand "the bottom layers exist in upper layers as well" at
> all and am completely lost on this part.
>
> Also, what do you mean by the "base of the print"? Is there a temple model
> with an extra flat base piece attached to it that you don't want to include
> in the print? I've certainly printed some with added bases so the pillars
> of entry areas, etc. don't break off, but I don't think any of the ones
> here have such bases attached.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/KD7LRJ/3D-LDS-Temples/discussions/12#discussioncomment-4505964>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A47B7I74EXBLFWRIWENMHWLWPOYYFANCNFSM6AAAAAATKS6YVU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
--David Engel

**Comment by hlyman** [comment]
I am sorry that I cannot help you.  I have downloaded the model from Thingiverse and I cannot find an gap with the bottom layer set to "0".  I use Simiplify3D as my slicer. 
What layer do you find gaps.
Try the file in Thingiverse.

Hugh Lyman
***@***.***
M: 253-569-9596

**Comment by LoopBak01** [comment]
Sorry, I'm not trying to imply there is a problem with the print. I'm just
trying to figure out a solution to a particular situation when I remove the
bottom layers. I've tried the GitHub and Thingiverse files. It's not a
specific problem to the Portland temple, I've noticed this on many of the
temples. "Problem" is not the correct word to use here either, as I don't
think it's an issue with the STL, it's a Cura settings/slicing issue I
believe. If you preview layer 97 and look at the space between the
overhangs and the walls on the "side" of the temple, with 0 bottom layers,
you'll see what I mean. I'm not sure file attachments will work here so
I've collected some screenshots. I hope you can view here to see what I'm
talking about.
<https://drive.google.com/file/d/1MemrdVddcqsAlpVLU12j_zsfUaoACX8C/view?usp=share_link>
Two screenshots show the "bottom layers" missing at layer 97 when that
setting is set to 0, and a screenshot that shows the "bottom layers"
present (yellow) at layer 97 when the setting is the default 4.
